deployment/terraform/variables.tf

```diff
@@ -273,3 +273,6 @@ variable "aws_lambda_service_role_policy_arn" {
 
 variable "oar_client_key" {}
 
+variable "aws_cloudfront_canonical_user_id" {
+  default = "c4c1ede66af53448b93c283ce9448c4ba468c9432aa01d700d3878632f77d2d0"
+}
```

Comment:
We need to hardcode this magic ID until https://github.com/hashicorp/terraform-provider-aws/issues/12512 is merged.

Reply:
👍 
We should consider opening an issue in the backlog so we have something to remind us to clean this up.

Reply:
Tracked in #1209!